src/integration/pages/mediaAssetPage/crossPlatformTests.js

```diff
@@ -23,23 +23,41 @@ export default () => {
     expect(timestampEl.textContent).toMatchSnapshot();
   });
 
-  const bulletedListEl = document.querySelector('main ul[role="list"]');
+  const bulletedListEl = document.querySelector('main ul[role="list"] > li');
 
   if (bulletedListEl) {
-    it('I can see the bulleted list item', () => {
+    it('I can see a bulleted list item', () => {
       expect(bulletedListEl).toBeInTheDocument();
       expect(bulletedListEl.textContent).toBeTruthy();
       expect(bulletedListEl.textContent).toMatchSnapshot();
     });
   }
 
-  const releatedContentEl = document.querySelector('section [role="list"]');
+  const bulletedListlink = document.querySelector(
+    'main ul[role="list"] > li > a',
+  );
 
-  if (releatedContentEl) {
-    it('I can see the related content', () => {
-      expect(releatedContentEl).toBeInTheDocument();
-      expect(releatedContentEl.textContent).toBeTruthy();
-      expect(releatedContentEl.textContent).toMatchSnapshot();
+  if (bulletedListlink) {
+    it('I can see a bulleted list item with link', () => {
+      expect(bulletedListlink.getAttribute('href')).toMatchSnapshot(
+        bulletedListlink.textContent,
+      );
+    });
+  }
+
+  const relatedContentItems = document.querySelectorAll(
+    'section [role="list"] a',
```

Comment:
To make this more specific and less likely to need updating in the future, I think we should add a 'data-e2e' attribute on the RelatedContent section element in the code, so in future if there are multiple sections, this will still refer to the Related Content section explicitly. 
```suggestion
    '[data-e2e="related-content"] a',
```

Reply:
What if we were to use the aria-labelled-by for the moment (ideally we would be using the data-e2e in lots of places):
![image](https://user-images.githubusercontent.com/58214768/80512959-3bc2f480-8976-11ea-9938-0038e10c5b05.png)

Reply:
This works: https://github.com/bbc/simorgh/pull/6300/commits/238fb43e0cf5f3ecd8098712d6647bf31723b9dc - what do you think?

Reply:
I have now added the data-e2e attribute to make testing this easier